scripts/bundle-rhmi-operators.sh

```diff
@@ -27,19 +30,24 @@ VERSIONS="${BUNDLE_VERSIONS:-$LATEST_VERSION}"
 ROOT=$(pwd)
 INDEX_IMAGE=""
 CATALOG_SOURCE_INSTALL="${OC_INSTALL:-true}"
+BUNDLE_ONLY="${BUNDLE_ONLY:-false}"
 
 
 start() {
-  clean_up
-  create_work_area
-  copy_bundles
-  check_upgrade_install
-  generate_bundles
-  generate_index
-  if [ "$CATALOG_SOURCE_INSTALL" = true ] ; then
-  create_catalog_source
+  if [ BUNDLE_ONLY ]; then
```

Comment:
You may want to change this if statement to keep it consistent with the rest of the file. 
```suggestion
  if [ "$BUNDLE_ONLY" = true ]; then
```

Reply:
Good point, thank you!